bee-storage/src/lib.rs

```diff
@@ -1,388 +1,2 @@
-extern crate bee_bundle;
-extern crate rand;
-
-pub mod sqlx;
 pub mod storage;
```

Comment:
I would not publicly expose the modules. Privately expose them and only publicly export what you really want to be used outside. But we can do that later if you prefer.

Reply:
will do now